include/jemalloc/internal/tsd_win.h

```diff
@@ -78,6 +96,12 @@ tsd_boot0(void) {
 	}
 	_malloc_tsd_cleanup_register(&tsd_cleanup_wrapper);
 	tsd_wrapper_set(&tsd_boot_wrapper);
+#if JEMALLOC_WIN32_TLSGETVALUE2
+	tgv2_mod = LoadLibraryA("api-ms-win-core-processthreads-l1-1-8.dll");
```

Comment:
I'm a bit concerned that this may not be forward compatible, e.g. the dll gets renamed or updated and we'd lose the support here (but in a non obvious way). Is there a different way that can detect version >= Windows 11, 24H2?

Reply:
yes, this is forward compatible as all future windows should support this API set api-ms-win-core-processthreads-l1-1-8 and loading the API set dll instead of a forwarded concrete dll (e.g., kernel32.dll) is the preferred way since kernel32.dll might not exist on other Windows SKUs. Usually, we don't recommend checking windows versions.